web/src/components/dashboards/AddDashboard.vue

```diff
@@ -187,6 +187,9 @@ export default defineComponent({
         delete this.dashboardData.id;
 
         if (dashboardId == "") {
+          // FIXME: Dashboard ID will be generated by the back end;
+          // vvv is working on that. Leave this field unset or set it
+          // to a dummy value (0).
```

Comment:
🚧 `dashboard_id` is not generated at the server side yet. Note the ‘To Do’ section in the PR description.